extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/BridgeHandler.java

```diff
@@ -187,76 +250,78 @@ private boolean checkLoginConfig(Config config) {
     }
 
     private Config loadAndCheckConfig() {
-        Configuration thingConfig = super.getConfig();
-        Config config = loadAndCheckConnectionData(thingConfig);
-        if (config == null) {
-            return null;
-        }
-        logger.info("Loading configuration");
-        ArrayList<String> numberExc = null;
-        // Parameters can't be null, because of an existing default value.
-        if (StringUtils
-                .isNotBlank(thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString())) {
-            try {
-                config.setSensordataRefreshInterval(Integer.parseInt(
-                        thingConfig.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString() + "000"));
-            } catch (NumberFormatException e) {
+        try {
+            Configuration thingConfig = super.getConfig();
+            Config config = loadAndCheckConnectionData(thingConfig);
+            if (config == null) {
+                return null;
+            }
+            logger.info("Loading configuration");
+            ArrayList<String> numberExc = null;
```

Comment:
Because it is only needed to build the error message for the thing status and, if everything is right, the ArrayList is not needed (null). At line 306 the thing status will be build, if it is not null. That the error appears is very improbable, because the parameter has to be a string.

But we can check to an empty list, too, if you think that this is better.

Reply:
> But we can check to an empty list, too, if you think that this is better.

Yes, please.